src/test/java/org/elasticsearch/bwcompat/OldIndexBackwardsCompatibilityTests.java

```diff
@@ -70,6 +84,76 @@ public static void initIndexes() throws Exception {
         Collections.sort(indexes);
     }
     
+    @AfterClass
+    public static void tearDownStatics() {
+        indexes = null;
+        indicesDir = null;
+    }
+    
+    @Override
+    public Settings nodeSettings(int ord) {
+        return ImmutableSettings.builder()
+            .put(Node.HTTP_ENABLED, true) // for _upgrade
+            .put(MergePolicyModule.MERGE_POLICY_TYPE_KEY, NoMergePolicyProvider.class) // disable merging so no segments will be upgraded
+            .build();
+    }
+    
+    void setupDataLoadingNodes() throws Exception {
```

Comment:
I named this method plural with future enhancements in mind. My plan is to actually start 2 nodes, one with a single data path, and another with 2 data paths.  loadIndex can then randomly choose to put the index on the single data path node, or split the index files across the 2 data paths node.  This will help test the upgrade path for the behavior change to remove the DistributorDirectory.